pages/workouts/index.js

```diff
@@ -32,28 +33,29 @@ export default function WorkoutsPage({
           <br />
           workouts
         </Headline>
-        <ButtonsSection>
-          <CreateLink href={"/workouts/create"}>
-            <Image alt="Edit" width={30} height={30} src="/icons/plus.svg" />
-            Create Workout
-          </CreateLink>
-          <FavouriteButton
-            type="button"
-            onClick={() => setisFavouritesMode(!isFavouritesMode)}
-          >
-            <FavouriteIcon
-              alt="Favourite"
-              width={35}
-              height={35}
-              src={
-                isFavouritesMode
-                  ? "/icons/star-filled-white.svg"
-                  : "/icons/star-white.svg"
-              }
-            />
-          </FavouriteButton>
-        </ButtonsSection>
+        <Login />
```

Comment:
Maybe you can render the Login component inside your Layout component. Then you don't have to render it on every site. It's up to you.